### PR TITLE
 [#1596] custom drag n drop label text

### PR DESCRIPTION
**Pull request by aluarius**
close: #1596 

## Description
Added optional props for custom drag'n'drop label text.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

**Review by m0ksem — COMMENTED** [review]
I think we also want to care about "Upload file" text here. Also, would be nice to have slots here, maybe. 

https://github.com/epicmaxco/vuestic-ui/blob/ec2c65ce510f285606f381573bbb63e4baa1c33e/packages/ui/src/components/va-file-upload/VaFileUpload.vue#L14-L25

**Comment by aluarius** [comment]
> I think we also want to care about "Upload file" text here. Also, would be nice to have slots here, maybe.

Agree, slots looks fine here.

**Review by Derranion — APPROVED** [review]
Thanks,

Could you please add this to the docs description.

Because otherwise users will not know how to use it without checking source code.

[Here are some slots examples in our docs](https://vuestic.dev/en/ui-elements/select#slots).

File: packages/ui/src/components/va-file-upload/VaFileUpload.vue [hunk 1]
```diff
@@ -9,7 +9,7 @@
         class="va-file-upload__field__text"
         v-if="dropzone"
       >
-        Drag’n’drop files or
+        <slot name="dropZoneLabel">Drag’n’drop files or</slot>
```
Comment (m0ksem) [inline packages/ui/src/components/va-file-upload/VaFileUpload.vue:12]: Having prop for label is more appropriate. We can use [Components Config]( https://vuestic.dev/en/services/components-config) for i18n for all components :) 

When I spoke about slot, I meant a slot that can be used to change overall file-upload visual (to replace default button and label text). Sorry for that. Let's better have props that can be changed in global config.

**Review by m0ksem — CHANGES_REQUESTED** [review]

**Comment by aluarius** [comment]
Added props for custom upload button & drop zone. Added slot for the whole upload area (no v-bind for events needed, cause there is hidden input, covering the whole area). Added examples and docs description.

**Review by m0ksem — APPROVED** [review]
Pretty nice now 😊

**Review by Derranion — APPROVED** [review]

**Pull request merged**